### PR TITLE
👕  Set up Standard JS linter

### PR DESCRIPTION
**Pull request by jasonrudolph**
Configure linter for [JavaScript Standard Style](https://standardjs.com) to foster consistency with the [Atom style guide](https://github.com/atom/atom/blob/572aec74d9fb612849a2b02cb5e54d515341c0fd/CONTRIBUTING.md#javascript-styleguide).

### TODO

- [x] Set up [JavaScript Standard Style](https://standardjs.com) linter
- [x] Resolve existing linter violations

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,5 +60,17 @@
       "default": "mt1",
       "order": 3
     }
+  },
+  "standard": {
+    "globals": [
+      "atom",
+      "setup",
```
Comment (50Wliu) [inline package.json]: Consider adding the `mocha` env instead of specifying all the mocha globals?

Comment (jasonrudolph) [reply package.json]: @50Wliu: That sounds promising. :smile: Can you point me toward an example?

Comment (Arcanemagus) [reply package.json]: Exactly like is done [here](https://github.com/atom/atom/blob/572aec74d9fb612849a2b02cb5e54d515341c0fd/package.json#L178-L189), but with `"mocha": true`.

Note that `mocha` is a [predefined environment](https://eslint.org/docs/user-guide/configuring#specifying-environments).

Comment (as-cii) [reply package.json]: Nice, yeah I think we should use the `mocha: true` environment.

Comment (jasonrudolph) [reply package.json]: Thanks! db56166f6510e04df6b541642c3c108b2da21fd8 adds the `"mocha": true` env config.

**Comment by jasonrudolph** [comment]
@as-cii: Thanks for the review. I pushed up several commits to address the linter violations in the `test` dir. Would you mind reviewing those changes?

**Pull request merged**